examples/electron-asar/app.js

```diff
@@ -2,7 +2,7 @@
 'use strict'
 
 const { app, ipcMain, BrowserWindow } = require('electron')
-const ipfsd = require('ipfsd-ctl')
+const ipfsd = require('../..')
```

Comment:
You can do `ipfsd-ctl` and then on the package.json of this example do `ipfsd-ctl: "file:/../.."`. See https://github.com/ipfs/js-ipfs/blob/master/examples/exchange-files-in-browser/package.json#L16